tests/Support/ConfigFixture.php

```
@@ -0,0 +1,94 @@
+			self::aboveRootGuardPath(),
+		];
+
+		foreach ( $paths as $path ) {
+			if ( is_file( $path ) ) {
+				chmod( $path, 0777 );
+				unlink( $path );
+			}
+		}
+
+		// Remove any .tmp.* artifacts near targets.
+		foreach ( [ ABSPATH, dirname( rtrim( ABSPATH, '/\\' ) ) ] as $dir ) {
```

PHPStan (2.2.5)

[error] Constant ABSPATH not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)